esx_service/vmdk_ops.py

```diff
@@ -705,7 +705,7 @@ def get_vol_path(datastore, tenant_name=None):
 
     if not os.path.isdir(dock_vol_path):
         # The osfs tools are usable for DOCK_VOLS_DIR on all datastores
-        cmd = "{0} {1}".format(OSFS_MKDIR_CMD, dock_vol_path)
+        cmd = "{0} '{1}'".format(OSFS_MKDIR_CMD, dock_vol_path)
```

Comment:
nit (since the line is touched anyways) - it could be `cmd = "{} '{}'".format(OSFS_MKDIR_CMD, dock_vol_path)` , no need for positionals, just sequential params are fine